hw/ip/rv_core_ibex/doc/_index.md

```diff
@@ -77,6 +77,7 @@ In general, when the CPU encounters an error, it is software's responsibility to
 However, there are situations where it may not be possible for software to collect any error logging.
 These situations include but are not limited to:
 * A hung transaction that causes watchdog to expire.
+* A double fault that caues the processor to stop execution.
```

Comment:
@matutem please have a look. Let me know if this helps explain things or if I should add more.